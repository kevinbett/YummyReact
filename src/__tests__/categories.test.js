import React from 'react';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';

import CategoryPost from '../categories/categories';

describe('CategoryPost Component', () => {
    const wrapper = shallow(<CategoryPost />);
    it('renders without crashing', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
    it('renders state initially', () => {
        expect(wrapper.state().name).toEqual('');
    })
});