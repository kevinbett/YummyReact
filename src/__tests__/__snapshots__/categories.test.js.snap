// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryPost Component renders without crashing 1`] = `
<div>
  <form
    onSubmit={[Function]}
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <TextField
      disabled={false}
      floatingLabelFixed={false}
      floatingLabelText="Category name"
      fullWidth={false}
      hintText="Enter category name"
      multiLine={false}
      onChange={[Function]}
      rows={1}
      type="text"
      underlineShow={true}
    />
    <br />
    <FlatButton
      disabled={false}
      fullWidth={false}
      label="Submit"
      labelPosition="after"
      labelStyle={Object {}}
      onClick={[Function]}
      onKeyboardFocus={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchStart={[Function]}
      primary={false}
      secondary={true}
    />
  </form>
</div>
`;
